<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Configuration</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Architecture</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="archComboBox">
     <item>
      <property name="text">
       <string>armeabi-v7a</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>arm64-v8a</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Threshold</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="thresholdSpinBox">
     <property name="toolTip">
      <string>Malloc stacktrace collect threshold.</string>
     </property>
     <property name="suffix">
      <string> byte</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>1048576</number>
     </property>
     <property name="value">
      <number>1024</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>libraries</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QListWidget" name="libraryListWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
